pkg/resources/view.go

```diff
@@ -194,7 +194,8 @@ var viewSchema = map[string]*schema.Schema{
 				},
 			},
 		},
-		Description: "If you want to change the name of a column or add a comment to a column in the new view, include a column list that specifies the column names and (if needed) comments about the columns. (You do not need to specify the data types of the columns.)",
+		Description:      "If you want to change the name of a column or add a comment to a column in the new view, include a column list that specifies the column names and (if needed) comments about the columns. You do not need to specify the data types of the columns. If this field is not specified, columns are inferred from the `statement` field by Snowflake.",
+		DiffSuppressFunc: IgnoreNewEmptyList([]string{"column_name"}),
```

Comment:
nit: whenever a list is needed to be passed, imo, it's better to have `...string` instead of `[]string`. Not a big deal, so we can leave it as is. It's more of a preference thing.

Reply:
I agree, but it's very inconsistent across the codebase. But changed to variadic arg.